src/Ansatz/MPS.jl

```diff
@@ -62,6 +62,30 @@ function MPS(arrays::Vector{<:AbstractArray}; order=defaultorder(MPS))
     return MPS(ansatz, NonCanonical())
 end
 
+MPS(arraysdims::NTuple{N,<:Vector{Int}}; order=defaultorder(MPS)) where {N} = MPS(collect(arraysdims); order=order)
+MPS(arraysdims::Vector{<:Tuple{Vararg{Int}}}; order=defaultorder(MPS)) = MPS(collect.(arraysdims); order=order)
+function MPS(arraysdims::NTuple{N,Tuple{Vararg{Int}}}; order=defaultorder(MPS)) where {N}
+    return MPS(collect(collect.(arraysdims)); order=order)
+end
+function MPS(arraysdims::Vector{<:Vector{Int}}; order=defaultorder(MPS))
+    @assert length(arraysdims[1]) == 2 "First array must have 2 dimensions"
+    @assert all(==(3) ∘ length, arraysdims[2:(end - 1)]) "All arrays must have 3 dimensions"
+    @assert length(arraysdims[end]) == 2 "Last array must have 2 dimensions"
+    issetequal(order, defaultorder(MPS)) ||
+        throw(ArgumentError("order must be a permutation of $(String.(defaultorder(MPS)))"))
+
+    return MPS(
+        map(arraysdims) do arrdims
+            mindim = minimum(arrdims)
+            arr = zeros(ComplexF64, arrdims...)
+            deltas = ntuple(x -> ntuple(_ -> x, length(arrdims)), mindim)
+            broadcast(delta -> arr[delta...] = 1.0, deltas)
+            arr
+        end;
+        order=order,
+    )
+end
+
```

Comment:
I would not create another constructor for `MPS`, since that may be misleading to the user who wants to directly create an `MPS` with some specific arrays. Instead, you could extend the function `ones` or one similiar, to make that when we call `ones(MPS, arraydims ... )` we get this kind of `MPS`. Something similar to the things we do with `rand`.

Reply:
As I said above, I agree it is better to not implement another constructor. The option that fits the best for me is `Base.identity`, check it out :)

Reply:
Yeah, I think `ones` can be misleading since it could mean you're initializing the MPS to a $\ket{11\dots 1}$ state.

I think `identity` is better because contracting against another "identity" MPS gives you 1, all the tensors are just COPY-tensors and "identity initialization" is how is known in some packages (it's also known as `eye`-initialization but `eye` is how the identity is known on those languages).